api/services/get.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+var self = get;
+module.exports = self;
+
+var async = require('async');
+var _ = require('underscore');
+var fs = require('fs');
+var path = require('path');
+
+var configHandler = require('../../common/configHandler.js');
+
+function get(req, res) {
+  var bag = {
+    reqQuery: req.query,
+    resBody: {},
+    initializeDefault: false,
+    component: 'services',
+    defaultService: {}
+  };
+
+  bag.who = util.format('services|%s', self.name);
+  logger.info(bag.who, 'Starting');
+
+  async.series([
+    _checkInputParams.bind(null, bag),
+    _get.bind(null, bag),
+    _getServicesJson.bind(null, bag),
+    _setDefault.bind(null, bag)
```

Comment:
the `_get` should be called at the end. it should return values from database after they've been set. so the workflow should be something like 
- check if db has any values
- if not, set default values
- return whatever is in the database
subsequent calls to this route should skip steps 1 and 2 and just get value from db